core/src/test/java/io/questdb/test/std/str/Utf8sTest.java

```diff
@@ -55,6 +64,15 @@ public void testContainsAscii() {
         Assert.assertFalse(Utf8s.containsAscii(Utf8String.EMPTY, "foobar"));
     }
 
+    @Test
```

Comment:
it would be good to have an upper case in "haystack" (the first argument of `containsLowerCaseAscii()`). 

otherwise an impl of `containsLowerCaseAscii()` which just delegates to `contains()` would pass all these test cases.

Reply:
Good point. Addressed in e7a465cb692f2ceef02b90b560fc7b0dd3a4eb3f